script/metric/step_5_import_metric_value.rb

```diff
@@ -2,6 +2,15 @@
 require File.expand_path('../../../config/environment',  __FILE__)
 require 'colorize'
 
+def is_valid_new_metric? metric_name, company, year
```

Comment:
Use def with parentheses when there are parameters.<br>Rename `is_valid_new_metric?` to `valid_new_metric?`.